allennlp/data/dataset_readers/dataset_reader.py

```diff
@@ -233,6 +233,14 @@ def apply_token_indexers(self, instance: Instance) -> None:
         """
         If `Instance`s created by this reader contain `TextField`s without `token_indexers`,
         this method can be overriden to set the `token_indexers` of those fields.
+
```

Comment:
```suggestion

        E.g. if you have you have `"source"` `TextField`, you could implement this method like this:

        ```python
        def apply_token_indexers(self, instance: Instance) -> None:
            instance["source"].token_indexers = self._token_indexers
        ```

```